src/values/metadata_value.rs

```diff
@@ -40,6 +40,8 @@ pub const FIRST_CUSTOM_METADATA_KIND_ID: u32 = 30;
 pub const FIRST_CUSTOM_METADATA_KIND_ID: u32 = 31;
 #[cfg(feature = "llvm13-0")]
 pub const FIRST_CUSTOM_METADATA_KIND_ID: u32 = 31;
+#[cfg(feature = "llvm14-0")]
+pub const FIRST_CUSTOM_METADATA_KIND_ID: u32 = 31;
```

Comment:
I'm not sure this is the correct value - how do I obtain and verify this constant?

Reply:
See the docs for the method

Reply:
Thanks, I have confirmed that the value is correct :+1:

I also added documentation but unfortunately it doesn't get picked up by `rustdoc` because of `#[cfg(feature = ...)]`. So the rustdoc comment either has to be duplicated for each LLVM version or fixed separately when https://github.com/rust-lang/rust/issues/43781 is stabilised.

Reply:
Could you link the ticket here in a non doc comment?

Reply:
How does that look?